tests/Unit/Scraper/Application/ConfiguratorTest.php

```diff
@@ -209,11 +212,10 @@ public function whenTryToFindNewXpathButNotFoundItShouldLogItAndResetVariant():
 
         Log::shouldReceive('warning')
             ->with("Field ':field-2:' with value ':value-2:' not found for ':scrape-url:'.");
-
-        $this->expectsEvents(ConfigurationScraped::class);
-
+            
         try {
             $this->configurator->configureFromDataset($posts);
+            Event::assertDispatched(ConfigurationScraped::class);
```

Comment:
This is dead code. Review other tests with the same behavior please.

Reply:
We have tested it that when an event is to be triggered then it returns true.